service_container/autowiring.rst

```diff
@@ -363,9 +363,40 @@ If you register this as a service, you now have *two* services that implement th
 which one to use. Remember, autowiring isn't magic; it looks for a service
 whose id matches the type-hint. So you need to choose one by creating an alias
 from the type to the correct service id (see :ref:`autowiring-interface-alias`).
+Additionally, you can define several named aliases if you want to use
+one implementation in some cases, and another implementation in some
+other cases.
 
-If you want ``Rot13Transformer`` to be the service that's used for autowiring, create
-that alias:
+
+For instance, you may want to use by default the ``Rot13Transformer``
+implementation when the ``TransformerInterface`` interface is type
+hinted, but use the ``UppercaseTransformer`` implementation in some
+specific cases.  To do so, you can create a normal alias from the
+``TransformerInterface`` interface to ``Rot13Transformer,`` and then
+create a *named alias* from a special string containing the interface
+followed by a variable named you will have to use when doing the
+injection::
+
+    namespace App\Service;
```

Comment:
Can you find another namespace as Service is a bit vague and promoting it in the official docs is not the best IMO

Or ist it still used in earlier examples?

I am currently on a phone 📱

Reply:
It's in the earlier examples, and besides, you know me, you know I wouldn't do that :P

Reply:
Fine for me then and yes, that’s why I asked you 👌🏻😁